src/style.css

```diff
@@ -0,0 +1,164 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+body,
+h2,
+h3 {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.foto_image {
+  width: 160px;
+  height: 134px;
+  margin-left: 19px;
+  margin-right: 19px;
+  margin-top: 32px;
+}
+
+.title {
+  width: 166px;
+  height: 36px;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: #060b35;
```

Comment:
Move all repeatable colors to CSS variables.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)